plusdeck/jupyter.py

```diff
@@ -102,31 +102,31 @@ def on_play_b(button):
 
     @stop.on_click
     def on_stop(button):
-        client.send(Command.Stop)
+        client.send(Command.STOP)
 
     eject = widgets.Button(value=False, description="⏏️", tooltip="Eject")
 
     @eject.on_click
     def on_eject(button):
-        client.send(Command.Eject)
+        client.send(Command.EJECT)
 
     fast_forward = widgets.Button(value=False, description="⏩", tooltip="Fast-Forward")
 
     @fast_forward.on_click
     def on_fast_forward(button):
-        client.send(Command.FastForward)
+        client.send(Command.FAST_B)
 
-    rewind = widgets.Button(value=False, description="⏪", tooltip="Rewind")
+    rewind = widgets.Button(value=False, description="⏪", tooltip="Fast Forward")
```

Comment:
I think this is a fix, but I'd like to check. If it *is* a fix, then it should probably be "Fast-Forward" for consistency?

Reply:
Good thing I checked, because I was actually getting confused by ipywidgets' odd idioms. Hah!